packages/emitter-framework/src/typescript/components/type-declaration.tsx

```diff
@@ -17,7 +17,7 @@ export type WithRequired<T, K extends keyof T> = T & { [P in K]-?: T[P] };
 export function TypeDeclaration(props: TypeDeclarationProps) {
   const { $ } = useTsp();
   if (!props.type) {
-    const refkeys = declarationRefkeys(props.refkey, props.name);
+    const refkeys = declarationRefkeys(props.refkey, props.type);
```

Comment:
Since `props.type` will be undefined (or otherwise falsy), should we keep with the `props.name` or even just nothing `declarationRefkeys(props.refkey)`  so we always get a fresh refkey? Not sure what makes sense here

Reply:
yeah just props.refkey seems appropriate here.